src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -122,8 +122,14 @@ public function send(string $body, array $headers, int $delay = 0): string
             $body,
             json_encode($headers),
             $this->configuration['queue_name'],
-            self::formatDateTime($now),
-            self::formatDateTime($availableAt),
+            $now,
+            $availableAt,
+        ], [
+            null,
+            null,
+            null,
+            Type::getType($this->getDateTimeType()),
```

Comment:
you don't need to call `Type::getType`. it should be enough to pass the string constant like it's done in other places https://github.com/symfony/symfony/blob/1e04345fd53b577141f57fe9df25ab9b6a249513/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L285